src/rgw/rgw_bucket.cc

```diff
@@ -1452,7 +1452,7 @@ static int bucket_stats(RGWRados *store, const std::string& tenant_name, std::st
   ::encode_json("owner", bucket_info.owner, formatter);
   formatter->dump_string("ver", bucket_ver);
   formatter->dump_string("master_ver", master_ver);
-  formatter->dump_stream("mtime") << ut;
+  ut.gmtime(formatter->dump_stream("mtime"));
```

Comment:
wierd locution

Reply:
agreed, but this is used elsewhere in rgw. both gmtime() and localtime() take a `std::ostream&`, the only difference is that `operator<<(utime_t)` calls localtime() directly

Reply:
ack, ok